src/NzbDrone.Core/Authentication/UserService.cs

```diff
@@ -13,12 +14,22 @@ namespace NzbDrone.Core.Authentication
 {
     public interface IUserService
     {
-        User Add(string username, string password);
+        User Add(string username, string password, UserRole role);
         User Update(User user);
         User Upsert(string username, string password);
         User FindUser();
         User FindUser(string username, string password);
         User FindUser(Guid identifier);
+        User UpdateByModel(User updatedUser);
+        public User FindUser(int id);
+
+        void Delete(int id);
+
+        public User FindUserFromApiKey(string apiKey);
+
+        List<User> All();
+
+        bool hasUsers();
```

Comment:
```suggestion
        bool HasUsers();
```

though as commented elsewhere we should have a default user instead of a wide open API with zero authentication.

Reply:
Perhaps generating a random initial password into the app.config would work (with the assumption that if you have access there then you're the superuser)